response/standard_handler.go

```diff
@@ -0,0 +1,128 @@
+package response
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/vesoft-inc/go-pkg/errorx"
+)
+
+const (
+	StandardHandlerBodyJson StandardHandlerBodyType = 0 // nolint:revive
+	StandardHandlerBodyNone StandardHandlerBodyType = 1
+)
+
+const (
+	standardHandlerFieldCode      = "code"
+	standardHandlerFieldMessage   = "message"
+	standardHandlerFieldData      = "data"
+	standardHandlerFieldDebugInfo = "debugInfo"
+)
+
+var _ Handler = (*standardHandler)(nil)
+
+type (
+	standardHandler struct {
+		params StandardHandlerParams
+	}
+
+	StandardHandlerBodyType int
+
+	StandardHandlerParams struct {
+		// GetErrCode used to parse the error.
+		GetErrCode func(error) *errorx.ErrCode
+		// CheckBodyType checks the type of body, default is StandardHandlerBodyJson.
+		CheckBodyType func(r *http.Request) StandardHandlerBodyType
+		// Errorf write the error logs.
+		Errorf func(format string, a ...interface{})
+		// DebugInfo add debugInfo details in body when error.
+		DebugInfo bool
+	}
+)
+
+func NewStandardHandler(params StandardHandlerParams) Handler {
+	return &standardHandler{
+		params: params,
+	}
+}
+
+func (h *standardHandler) GetStatusBody(r *http.Request, data interface{}, err error) (httpStatus int, body interface{}) {
+	httpStatus = http.StatusOK
```

Comment:
Why not use GetResponse? StatusBody seems to be like just returning one field

Reply:
`StatusBody` means `Status` and `Body`.

`GetResponse` is easily misunderstood as `http.Response`.